docs/install_guides/raspbian-classification-sample.md

```diff
@@ -0,0 +1,34 @@
+# Build and Run Image Classification Sample {#raspbian_classification_sample}
+
+@sphinxdirective
+
+.. _raspbian classification sample:
+
+@endsphinxdirective
+
+Follow the next steps to use the pre-trained image classification model using Inference Engine samples from the OpenVINO toolkit.
+
+1. Navigate to a directory that you have write access to and create a samples build directory. This example uses a directory named `build`:
+   ```sh
+   mkdir build && cd build
+   ```
+2. Build the Hello Classification Sample:
+   ```sh
+   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /opt/intel/openvino_2022/samples/cpp
```

Comment:
```suggestion
2. Download samples source code from https://github.com/openvinotoolkit/openvino/tree/master/samples/cpp
3. Build the Hello Classification Sample:
   ```sh
   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS="-march=armv7-a" /path/to/samples/cpp
```